Python/ast_opt.c

```diff
@@ -971,6 +726,119 @@ astfold_withitem(withitem_ty node_, PyArena *ctx_, _PyASTOptimizeState *state)
     return 1;
 }
 
+#define IS_CONST_EXPR(N) \
+    ((N)->kind == Constant_kind)
+
+#define CONST_EXPR_VALUE(N) \
+    ((N)->v.Constant.value)
+
+#define IS_COMPLEX_CONST_EXPR(N) \
```

Comment:
```suggestion
#define IS_CONST_COMPLEX_EXPR(N) \
```

Otherwise it reads as "is this a complex expression".